tools/packaging/kata-deploy/binary/src/k8s/nfd.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2019 Kata Containers community
+// Copyright (c) 2025 NVIDIA Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+use super::client as k8s;
+use crate::config::Config;
+use anyhow::Result;
+use log::info;
+use std::env;
+
+pub async fn setup_nfd_rules(config: &Config) -> Result<bool> {
+    let expand_runtime_classes_for_nfd = false;
```

Comment:
Given this is only ever false, have you considered just returning `OK(false)` for more clarity?